deploy/kessel-inventory.yaml

```diff
@@ -15,7 +15,7 @@ objects:
         - kessel-relations
       deployments:
         - name: api
-          replicas: 1
+          replicas: ${{REPLICAS}}
```

Comment:
Yep, its actually required because replicas is not a string field so you can't use single. IIRC, double brace just inserts the value without quotes this way it shows up as a number